.eslintrc.js

```diff
@@ -61,28 +46,12 @@ module.exports = {
         node: true,
       },
       plugins: ['node'],
-      rules: Object.assign(
-        {},
-        require('eslint-plugin-node').configs.recommended.rules,
-        {
-          // add your custom rules and overrides for node files here
-        }
-      ),
+      extends: ['plugin:node/recommended'],
     },
-
-    // test files
     {
```

Comment:
Not 100% sure what this entry was for? Since it's not part of the default blueprint, seems safe to delete.

Reply:
Sure